virttest/ip_sniffing.py

```diff
@@ -304,7 +304,7 @@ def is_supported(cls, session=None):
 
     @classmethod
     def _get_version(cls, session):
-        cmd = "%s --version" % cls.command
+        cmd = "%s -v" % cls.command
```

Comment:
Is there some reason have to do so?

Reply:
It looks like '--version' is not a valid option in 1.10.14.  but '-v' supports all versions.

# tshark --version
tshark: invalid option -- '-'
TShark 1.10.14 (Git Rev Unknown from unknown)

Reply:
I see, thanks.